defaults/main.yml

```diff
@@ -1,4 +1,4 @@
 ---
-postgresql_support_libpq_version: "9.5.*.pgdg14.04+2"
+postgresql_support_libpq_version: "9.5.*.pgdg*"
```

Comment:
Strange. All I did was make this edit to the role and re-run (did not manually install package.)

Is this package maybe already a dependency of something else DRIVER installs?